tests/ttnn/unit_tests/operations/test_moreh_linear.py

```diff
@@ -45,7 +149,7 @@ def moreh_linear(shapes, has_bias, has_output, compute_kernel_config, device):
     ## test for equivalance
     rtol = atol = 0.1
     cpu_layout = ttnn.ROW_MAJOR_LAYOUT
```

Comment:
Is this still being used anywhere? If not, you can remove it

Reply:
Resolved in latest commit